docs/developer-guide/background-linters.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: Setting up background linters
+section: Developer guide
+permalink: /docs/developer-guide/background-linters/
+---
+
+# Setting up background linters
+
+ESLint *must* be used when developing Origami components. It's useful to have linters run in the background and highlight errors as you're typing. SublimeLinter and ESLint packages are recommended for Sublime Text and JetBrains IDEs respectively.
+
+## Install dependencies
+
+If you already have Origami Build Tools (OBT) and ESLint configured, skip to the [Installing background linter plugins](#Installing-background-linter-plugins) section.
+
+Ensure you have installed [Origami Build Tools](https://www.npmjs.com/package/origami-build-tools). This will provide you with Sass, SCSS-lint and provide instructions on how to set up Node.js and Ruby. 
```

Comment:
Actually, better to go all lower case. The thing is, it's Scss and Sass, but the guys behind scss-lint write Sass wrong :P